### PR TITLE
Added revised codeql playbook

### PR DESCRIPTION
**Pull request by sand-hya**
Revised playbook for CodeQL run. Please review.

### CODE REVIEW

File: playbooks/besman-codeql_scan-playbook.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    echo -e "\n${BLUE}🛫 Preparing your machine for Code QL Run .${RESET}"
+
+    mkdir -p $HOME/$PROJECT_NAME
+    echo -e "ls -l $PWD/$PROJECT_NAME"
+}
+
+function __besman_execute {
+    # Generating CodeQL report 
+    curl -sS \
```
Comment (asa1997) [inline playbooks/besman-codeql_scan-playbook.sh:39]: @sand-hya You should have the actual steps to perform the assessment in a steps file.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@sand-hya Please make the required changes.

File: playbooks/besman-codeql_scan-playbook.sh (hunk 1 above)
Comment (sand-hya) [reply playbooks/besman-codeql_scan-playbook.sh:39]: @asa1997 will introduce the steps file and move this to steps file with modification

**Review by pramit-d — CHANGES_REQUESTED** [review]
@sand-hya Please add a version for playbooks. The file name should contain the version number.
Use [sbom generator](https://github.com/Be-Secure/besecure-playbooks-store/blob/main/playbooks/besman-spdx-sbom-generator-0.0.1-playbook.sh) playbook as an example and modify your playbook file accordingly.

**Pull request closed without merging**